man/scrot.txt

```diff
@@ -2,128 +2,135 @@ NAME
+    - It is possible to capture a specific window or a rectangular area on the
+      screen.
 
-  scrot also can be used to monitor a desktop PC in admin absent
-  and register unwanted activities.
+  Because scrot is a command line utility, it can easily be scripted and put to
+  novel uses. For instance, scrot can be used to monitor an X server in absence.
+
+  scrot is free software under the MIT-advertising license.
 
 OPTIONS
-  -h, --help         Display help output and exit.
-  -v, --version      Output version information and exit.
-  -D, --display      Specify the display to use; see X(7).
-  -a, --autoselect   Non-interactively choose a rectangle of x,y,w,h.
-  -b, --border       When selecting a window, grab wm border too.
-                     Use with --select to raise the focus of the window.
-  -c, --count        Display a countdown when used with delay.
-  -d, --delay NUM    Wait NUM seconds before taking a shot.
-  -e, --exec APP     Exec APP on the saved image.
-  -q, --quality NUM  Image  quality (1-100) high value means high size, low
-                     compression. Default: 75. (Effect differs depending on
-                     file format chosen).
-  -m, --multidisp    For multiple heads, grab shot from each and join them
-                     together.
-  -s, --select       Interactively select a window or rectangle
-                     with the mouse (use the arrow keys to resize).
-                     See -l and -f options.
-  -l, --line         Indicates the style of the line when the -s option is used.
-                     See SELECTION STYLE.
-  -f, --freeze       Freeze the screen when the -s option is used.
-  -u, --focused      Use the currently focused window.
-  -t, --thumb NUM|GEOM  Generate thumbnail too. NUM is the percentage of the
-                        original size for the thumbnail to be. Alternatively,
-                        a GEOMetry can be specified, example: 300x200.
-  -z, --silent       Prevent beeping.
-  -p, --pointer      Capture the mouse pointer.
-
-  -o, --overwrite    By default scrot does not overwrite the files, use this option to allow it.
-  -n, --note         Draw a text note. See NOTE FORMAT.
-  -k, --stack        Capture stack/overlapped windows and join them together.
-                     A running Composite Manager is needed.
-  -C, --class NAME   Window class name. Associative with options: -k.
-  -S, --script CMD   Imlib2 script commands.
+  -a, --autoselect X,Y,W,H  Non-interactively choose a rectangle starting at
+                            position X,Y and of W by H resolution.
+  -b, --border              When selecting a window, grab the WM's border too.
+                            Use with -s to raise the focus of the window.
+  -C, --class NAME          NAME is a window class name. Associative with -k.
+  -c, --count               Display a countdown when used with -d.
+  -D, --display             Specify the display to use; see X(7).
```

Comment:
This option requires an argument, maybe `-D, --display NAME`  and at the beginning   `scrot [...] -D [NAME]...`

Reply:
You're right.